.github/CODEOWNERS

```diff
@@ -1,16 +1,11 @@
 *                                       @percona/pmm-review-be
-/build/                                 @atymchuk @percona/pmm-review-be
-/admin/                                 @percona/pmm-review-be
```

Comment:
1/ removing myself due to the forthcoming name change
2/ `@percona/pmm-review-be` alone doesn't make sense because of the star